test/unit/org/apache/cassandra/cql3/KeywordTestBase.java

```diff
@@ -59,6 +61,12 @@ public KeywordTestBase(String keyword, boolean isReserved)
         this.isReserved = isReserved;
     }
 
+    @BeforeClass
+    public static void beforeClass()
+    {
+        StorageService.instance.setUpDistributedSystemKeyspaces();
```

Comment:
why is that change needed now?

Reply:
Well, previously the `CQLTester` simply removed the tables and views which it was aware of (that, which were created by `createTable`, `createView` methods, so that it recorded them as created by the test code and up to be removed after test. However, views in this case were created by generic `schemaChange` so those views were not recorded... and not removed after the test.

However, now we are removing all the stuff from the `KEYSPACE`, including all the views. Removing a view follows updating some table in the `system_distributed` keyspace, which is normally not created in CQLTester. Hence, this test started to fail that it cannot access that keyspace...